src/flownet/parameters/_relative_permeability.py

```diff
@@ -151,8 +149,17 @@ def __init__(
             distribution_values["parameter"].unique()
         )
         self._satnum: pd.DataFrame = satnum
-        self._swof, self._sgof = self._check_parameters()
         self._phases = phases
+        self._interpolation_values: Optional[pd.DataFrame] = None
+        self._scal_for_interp: Optional[PyscalList] = None
+        if isinstance(interpolation_values, pd.DataFrame):
```

Comment:
As `interpolation_values` is will be evaluated as `True` when it is a `pd.DataFrame` and `False` when it is `None`, you can drop the `isinstance()` call.

Reply:
If i drop the isinstance (which was what i did at first), i get 'ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().'

Reply:
OK - then ignore my comment - it's wrong in this case then.